performance/shopify/comment_form.rb

```diff
@@ -15,7 +15,7 @@ def initialize(tag_name, markup, options)
   def render(context)
     article = context[@variable_name]
 
-    context.stack do
+    context.stack('form') do
```

Comment:
Shouldn't `'form'` be frozen here?
```suggestion
    context.stack('form'.freeze) do
```